enterprise-ws-spec/src/main/asciidoc/Handlers.adoc

```diff
@@ -125,146 +110,95 @@ Common scenarios are logging, metering, and accounting.
+for processing inbound or outbound messages and invokes the lifecycle method annotated
+with jakarta.annotation.PreDestroy.
 
-Pooling of Handler instances is allowed, but is not required. If Handler
-instances are pooled, they must be pooled by Port component. This is
-because Handlers may retain non-client specific state across method
-calls that are specific to the Port component. For instance, a Handler
-may initialize internal data members with Port component specific
-environment values. These values may not be consistent when a single
-Handler type is associated with multiple Port components. Any pooled
-instance of a Port component’s Handler in a Method Ready state may be
-used to service the handleRequest(), handleResponse(), and handleFault()
-methods. It is not required that the same Handler instance service both
-the handleRequest() and handleResponse() or handleFault() method
-invocations of any given request.
-
-===== Handler Life Cycle with JAX-WS
-
-With JAX-WS the Handler lifecycle is similar to JAX-RPC illustration
-shown in the previous section with the following differences:
-
-* The init() method is now replaced by a method carrying a
-javax.annotation.PostConstruct annotation (defined by JSR-250
-specification, Section 2.4). This method is only called after all
-injections requested by the Handler are completed.
-* The destroy() method is replaced by a method carrying a
-javax.annotation.PreDestroy annotation (defined by JSR-250
-specification, Section 2.5).
-* Methods handleRequest() and handleResponse() are replaced by
-handleMessage().
-
-The lifecycle methods annotated with javax.annotation.PostConstruct and
-javax.annotation.PreDestroy in the Handler implementation allows the
+The lifecycle methods annotated with jakarta.annotation.PostConstruct and
+jakarta.annotation.PreDestroy in the Handler implementation allows the
 container to notify a Handler instance of impending changes in its
 state. Detailed requirements for Handler lifecycle are described in
-section 9.3.1 JAX-WS specification. The Handler may use the notification
+Jakarta XML Web Services specification. The Handler may use the notification
 to prepare its internal state for the transition. The container is
-required to call lifecycle methods annotated with
-javax.annotation.PostConstruct and javax.annotation.PreDestroy as
-described below.
+required to call lifecycle methods annotated with jakarta.annotation.PostConstruct 
+and jakarta.annotation.PreDestroy as described below.
 
 The container must carry out any injections (if any) requested by the
-handler, typically via the @Resource annotation (see section 2.2 of
-JSR-250 Common Annotations for Java Platform specification). After all
-the injections have been carried out, the container must invoke the
-method carrying a javax.annotation.PostConstruct annotation. This method
+handler, typically via the @Resource annotation (see Jakarta Annotations specification)
```

Comment:
missing dot after the end of the sentence

Reply:
Yes. I see it, will fix it.